website/public/js/controllers/settingsCtrl.js

```diff
@@ -66,12 +66,23 @@ habitrpg.controller('SettingsCtrl',
       User.set({'flags.newStuff':true});
     }
 
-    $scope.saveDayStart = function(){
-      var dayStart = +User.user.preferences.dayStart;
-      if (_.isNaN(dayStart) || dayStart < 0 || dayStart > 24) {
+    $scope.passDayStart = User.user.preferences.dayStart;
```

Comment:
Let me know if you think of something, and I'll chat with my cleverer half about it. I'd be happy to change it!